concept/type/AttributeType.java

```diff
@@ -38,6 +38,9 @@
     @Override
     Stream<? extends AttributeType> getSubtypes();
 
+    @Override
+    Stream<? extends AttributeType> getSubtypesDirect();
```

Comment:
We were going to rename these methods to `getSubtypesExplicit`, etc, right?

Reply:
Yes already renamed, 2 seconds after you start reviewing.